### PR TITLE
fixes #16503 - composite view ui - newer component version exists

### PR DESCRIPTION
**Pull request by bbuckingham**
Update the composite content view UI to let the user know
if a newer version of a component content view exists.

### CODE REVIEW

**Comment by bbuckingham** [comment]
The following is an example of the UI with the change:
newer_version

File: engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-composite-content-views-list.controller.js [hunk 1]
```diff
@@ -33,6 +33,13 @@ angular.module('Bastion.content-views').controller('ContentViewCompositeContentV
             return deferred.promise;
         };
 
+        $scope.newerVersionExists = function (contentViewVersion) {
+            var found = _.find(contentViewVersion['content_view']['content_view_version_ids'], function(num) {
```
Comment (jlsherrill) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-composite-content-views-list.controller.js]: With incremental update, the idea of a 'newer' version is somewhat muddled.  Is it by version #? or just last published?  

I would think it'd be by version number (i.e. 3.0 is newer than 2.2) even if the greater number was published before the lesser numbered version.

Comment (bbuckingham) [inline engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-composite-content-views-list.controller.js]: Good point.  It is actually using the database id, which implies the most recent version published.  In the example above, that would mean 2.2 is newer than 3.0.

**Comment by bbuckingham** [comment]
@jlsherrill, updated the PR to base latest off of major version.  Below is an updated screenshot for reference:

latest_version_based_on_major

**Comment by jlsherrill** [comment]
ACK

**Pull request merged**